smslibrary/build.gradle

```diff
@@ -37,5 +37,8 @@ dependencies {
     testImplementation 'org.mockito:mockito-core:1.10.19'
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
-    implementation 'com.github.CremaLuca:android-preferences:2.3'
+    implementation 'com.github.CremaLuca:android-preferences:3.0'
```

Comment:
You can remove this, it's already present 7 lines above.

Reply:
Ok, the repetition was introduced in commit https://github.com/Cogno-IDU/smslibrary/commit/b7b1e05e59f669d2a05b397adca07b27ad4319e8 though, not in this pull request

Reply:
Yup, saw it too, I've also removed it in my pr (#39 ), you can do it too if you want.